src/function/struct/struct_pack_function.cpp

```diff
@@ -33,6 +33,17 @@ void StructPackFunctions::compileFunc(FunctionBindData* /*bindData*/,
     }
 }
 
```

Comment:
Better clearly state the implicit assumption of `src` being 0 and `dst` being 1 here.

Reply:
I'll try to do this in a more principled way.